udata_front/theme/gouvfr/assets/js/components/search/MenuSearch/MenuSearch.vue

```diff
@@ -122,45 +121,23 @@ export default defineComponent({
     const showAndFocus = () => {
```

Comment:
Can you move this function to the composable too ?

Also, you can refactor the if to not include a `!`, it will be easier to read.

`searchSelectedOption` can be added as options to `useDropdown`.

This way, we can even pass `selectedOption` as argument to `searchSelectedOption` and remove `searchSelectedOption` from the composable export.